support-frontend/assets/pages/contributions-landing/components/ContributionThankYou/ContributionThankYouSupportReminder.jsx

```diff
@@ -194,13 +194,14 @@ const ContributionThankYouSupportReminder = ({
           <p>
             <span css={styles.hideAfterTablet}>
               Choose a time when we can invite you to support our journalism
-              again. We’ll send you a single email, with no obligation.
+              again. We’ll send you a maximum of two reminder emails, with no
+              obligation.
             </span>
             <span css={styles.hideBeforeTablet}>
-              Lots of readers chose to make single contributions at various
-              points in the year. Opt in to receive a reminder in case you would
-              like to support our journalism again. This will be a single email
-              with no obligation.
+              Many readers choose to support Guardian journalism by making
```

Comment:
Should it be "support the Guardian's journalism" ?

Reply:
This is how it is in the copy doc from Iona, I can check though?

Reply:
@ionamckendrick hey - can you weigh in on this one?

Reply:
@TomPretty @paulbrown1982 Sorry I was slow to reply here! This looks great. By the way "Guardian journalism" is a Kath/editorial fave, rather than the way you wrote it Paul. Neither are wrong by any stretch - but where space is tight, I use the former. Thanks! 👏